typescript/api-gateway-async-lambda-invocation/lib/api-gateway-async-lambda-invocation-stack.ts

```diff
@@ -0,0 +1,170 @@
+import * as cdk from 'aws-cdk-lib';
+import { AccessLogFormat, AwsIntegration, LambdaIntegration, LambdaRestApi, LogGroupLogDestination, MethodLoggingLevel } from 'aws-cdk-lib/aws-apigateway';
+import { AttributeType, Table } from 'aws-cdk-lib/aws-dynamodb';
+import { PolicyDocument, PolicyStatement, Role, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
+import { Code, Function, Runtime } from 'aws-cdk-lib/aws-lambda';
+import { LogGroup, RetentionDays } from 'aws-cdk-lib/aws-logs';
+import { Construct } from 'constructs';
+import path = require('path');
+
+export interface Properties extends cdk.StackProps {
+  readonly prefix: string;
+}
+
+export class ApiGatewayAsyncLambdaStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props: Properties) {
+    super(scope, id, props);
+
+    // DynamoDB table for job status
+    const jobTable = new Table(this, `${props.prefix}-table`, {
+      partitionKey: { name: 'jobId', type: AttributeType.STRING },
+      tableName: `${props.prefix}-job-table`,
+      removalPolicy: cdk.RemovalPolicy.DESTROY, // NOT recommended for production; Set `cdk.RemovalPolicy.RETAIN` for production
+    });
+
+    // Create a Log Group for API Gateway logs
+    const fnLogGroup = new LogGroup(this, `${props.prefix}-fn-log-group`, {
+      retention: RetentionDays.ONE_WEEK, // Customize the retention period as needed
+    });
+
+    //  create a lambda function
+    const jobHandler = new Function(this, `${props.prefix}-fn`, {
+      runtime: Runtime.NODEJS_20_X,
+      handler: 'job_handler.handler',
+      code: Code.fromAsset(path.join(__dirname, '../assets/lambda-functions')),
```

Comment:
Missing assets folder with the code for the lambda function.

Reply:
[job_handler.js](https://github.com/aws-samples/aws-cdk-examples/pull/1089/commits/26b5bc3ce716d8cddc785fcaee65ec28091d038c) has been added.